packages/cli/src/commands/generate/cell/cell.js

```diff
@@ -180,11 +153,26 @@ export const { command, description, builder, handler } =
         type: 'boolean',
       },
     },
-    includeAdditionalTasks: () => {
+    includeAdditionalTasks: ({ name: cellName }) => {
       return [
         {
           title: `Generating types ...`,
-          task: generateTypes,
+          task: async (_ctx, task) => {
+            const queryFieldName = nameVariants(
+              removeGeneratorName(cellName, 'cell')
+            ).camelName
+            const doesProjectHaveSdl = await checkProjectForQueryField(
+              queryFieldName
+            )
+
+            if (doesProjectHaveSdl) {
```

Comment:
This var named seemed a little wordy so tried to shorten it a bit:
```suggestion
            const projectHasSdl = await checkProjectForQueryField(
              queryFieldName
            )

            if (projectHasSdl) {
```

Reply:
👍